scripts/linters/test_files/valid.py

```diff
@@ -29,6 +29,7 @@
 from core.jobs.batch_jobs import exp_search_indexing_jobs            # pylint: disable=unused-import  # isort: skip
 from core.jobs.batch_jobs import model_validation_jobs               # pylint: disable=unused-import  # isort: skip
 from core.jobs.batch_jobs import opportunity_management_jobs         # pylint: disable=unused-import  # isort: skip
+from core.jobs.batch_jobs import question_migration_jobs                # pylint: disable=unused-import  # isort: skip
```

Comment:
Optional* Should we follow the the structure of `# pylint.. ` align vertically with other lines?

Reply:
Done